cmd/gitstrap/main.go

```diff
@@ -55,6 +61,33 @@ func main() {
 	}
 }
 
+func getToken(flagToken, savedToken string) string {
+	if flagToken != "" {
+		return flagToken
+	}
+	if savedToken != "" {
+		return savedToken
+	}
+	return ""
+}
+
+func savedToken(filePath string) string {
+	fmt.Println("Checking if there is a saved GitHub token at", filePath)
+	token, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		return ""
```

Comment:
@vassilevsky I think it would be more logical to return `error` if we can't read a token from file